types/aws-cloudfront-function/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
Why do you need this exemption? is it because the package doesn't have a corresponding npm package? Or is it because it is intended entirely for consumption from the browser, not node?

Reply:
It's because there's no package. It's basically AWS documented structure. It's the same as `aws-lambda` but for CloudFront functions.

Reply:
Removed